integration/emr/alluxio-emr.sh

```diff
@@ -300,13 +300,14 @@ main() {
 
   # Download files provided by "-f" to /opt/alluxio/conf
   IFS=" " read -ra files_to_be_downloaded <<< "${files_list}"
-  for file in "${files_to_be_downloaded[@]}"; do
-    local filename="$(basename ${file})"
-    download_file "${file}"
-    sudo mv "${filename}" "${ALLUXIO_HOME}/conf/${filename}"
-  done
-  sudo chown -R alluxio:alluxio "${ALLUXIO_HOME}/conf"
-
+  if [ ${#files_to_be_downloaded[@]} -gt 0 ]; then
```

Comment:
I would also quote these values just in case.

```
"${#files_to_be_downloaded[@]}" -gt "0"
```

Obviously no spaces are expected..but hey you never know

Reply:
done